src/components/clusterprovider/azure/azureclusterprovider.ts

```diff
@@ -446,3 +450,7 @@ function parseCluster(encoded: string) : azure.ClusterInfo {
         name: encoded.substr(delimiterPos + 1)
     };
 }
+
+async function refreshExplorer() : Promise<void> {
```

Comment:
remove async and replace await with return

Reply:
Hmm, as this is conceptually `void`, I kind of find the imperative `await` more natural than a `return` (though I know it is indeed actually returning a Promise/Thenable object rather than being truly void).  Would prefer to leave it unless you feel strongly.